packages/contracts/test/10_unit-testing/11_plugin.ts

```diff
@@ -233,6 +276,73 @@ describe('Multisig', function () {
     });
   });
 
+  describe('reinitialize', async () => {
+    it('reverts if trying to re-reinitializeFrom', async () => {
+      const {uninitializedPlugin, deployer} = await loadFixture(fixture);
+
+      const encodedDummyTarget = ethers.utils.defaultAbiCoder.encode(
+        ['address', 'uint8'],
+        [deployer.address, Operation.delegatecall]
+      );
+
+      // reinitialize the plugin.
+      await uninitializedPlugin.initializeFrom(
+        latestInitializerVersion,
+        encodedDummyTarget
+      );
+
+      // Try to reinitialize the  plugin.
+      await expect(
+        uninitializedPlugin.initializeFrom(
+          latestInitializerVersion,
+          encodedDummyTarget
+        )
+      ).to.be.revertedWith('Initializable: contract is already initialized');
+    });
+
+    it('reverts if trying to initializeFrom an initialized plugin', async () => {
+      const {initializedPlugin, deployer} = await loadFixture(fixture);
+
+      const encodedDummyTarget = ethers.utils.defaultAbiCoder.encode(
+        ['address', 'uint8'],
+        [deployer.address, Operation.delegatecall]
+      );
+
+      // Try to reinitialize the  plugin.
+      await expect(
+        initializedPlugin.initializeFrom(
+          latestInitializerVersion,
+          encodedDummyTarget
+        )
+      ).to.be.revertedWith('Initializable: contract is already initialized');
+    });
+
+    // todo add test for checking that plugins already initialized on a previous version can not be initialized again
+    it('reverts if trying to initialize lower version plugin');
```

Comment:
remove this test, is already done in https://github.com/aragon/multisig-plugin/blob/9164a7c3e382f5613464719dee84c70f78be4a59/packages/contracts/test/10_unit-testing/11_plugin.ts#L170-L184

Reply:
done in #22